source/reference/limitations.txt

```diff
@@ -48,6 +48,8 @@ General Limitations
 - The MongoDB Shared Tier is not supported.
 - The :data:`system.js <<database>.system.js>` collection is not
   replicated.
+- Collections with dollar (``$``) prefixed field names are not
```

Comment:
I'm fine with saying that $-prefixed fields will not be supported overall, though technically they should succeed during the CEA phase of replication. We won't be able to test the CEA-only behavior rigorously though.

CC @alan-zheng-mdb for confirmation

Reply:
Can we add a link to the ("$") DOCs?